docs/platform/deployservices.md

```diff
@@ -1,13 +1,37 @@
 # Deploy your platform using SmartSDK recipes
 
+This section will give you pointers to deploy core services of SmartSDK using
```

Comment:
This section provide a quick introduction to deploy some core FIWARE services using SmartSDK recipes. In particular, we will see how you can deploy:
- xxxx a service to blah blah.
-
-
 For more details and advanced topics, check the specific documentation here.